app/javascript/layouts/Sidebar/GetHelpSidebarBlock.tsx

```diff
@@ -15,7 +15,7 @@ const GetHelpSidebarBlock = () => (
     <LinkButton
       transition={true}
       newTab={true}
-      href={"https://www.sf.gov/resource/2022/homebuyer-program-counseling-agencies"}
+      href={getSfGovUrl("https://sf.gov/node/7209", 7209)}
```

Comment:
I understand that it is flexible, but the implied pattern from all previous items in the `SFGOV_LINKS` array is to use the plain-english URL, probably for better readability.

I'll downgrade this from an issue to a suggestion, but if it's not addressed in this PR I'll fix this myself if I happen to touch related code in one of my own future PRs.

Reply:
I think it's worth asking (Bridget?) if either of them might change independently - the node or the link. Because if they didn't, why would we have 2 arguments instead of just accepting the node? I tend to agree w/ jim here, and it's also an easy update to make!

Reply:
I can make the change for the 7209 PR.